ide/che-core-ide-stacks/src/main/resources/images/type-zend.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
@kaloyan-raev : everything looks good, except that you need to add a licence header in the SVG which is compliant with EPL. 

You can check what bitnami did for example: 
https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/src/main/resources/images/type-bitnami.svg